kubeflow/config.yaml

```diff
@@ -0,0 +1,121 @@
+# This config file specifies which Kubeflow components to deploy.
+# Each line has a relative path from this folder to manifests of a component to
+# deploy.
+# Each component may have a comment like: `dependencies: [ xxx ]`
+# It means this component depends on listed components.
+# Note, that the information may not be fully accurate at the moment. Welcome
+# feedback if you find any mistakes.
+
+# You can delete/comment lines to selectively pick the components you want to
+# deploy.
+# If you want to customize one component, edit or overlay manifests of that
+# component.
+components:
```

Comment:
It is a very good approach for component customization!

Should we call out that `the order of component list` here matters? Since some components depends on others, it will cause error if we swap the component order here. For example: cnrm should be deployed before any applciations.